src/main/java/ch/wisv/areafiftylan/utils/mail/MailServiceImpl.java

```diff
@@ -171,6 +171,13 @@ public void sendTeamInviteMail(User user, String teamName, User teamCaptain) {
     }
 
     @Override
+    public void sendSeatOverrideMail(User user) {
+        String subject = "Your seat was reset";
+        String message = "Your seat was reset by an admin.\nPlease log in to My Area to select your new seat.";
```

Comment:
I think this message is too generic and blunt. As this is quite exceptional and intrusive, this should be rewritten with some apology and reasoning